Doc/c-api/init.rst

```diff
@@ -1375,10 +1375,6 @@ All of the following functions must be called after :c:func:`Py_Initialize`.
    Reset all information in a thread state object.  The global interpreter lock
    must be held.
 
-   .. versionchanged:: 3.9
```

Comment:
I would prefer to keep this versionchanged and add a new one for Python 3.13 instead, to keep the full history.